performance/liberty/configs/benchmark.sh

```diff
@@ -0,0 +1,79 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+echo "***** Running Benchmark Script *****"
+
+currentDir="$(pwd)"
+echo "$currentDir"
+echo "Current Dir: $PWD"
+
+#TODO: Remove these once the use of STAF has been eliminated from all the benchmark scripts
+export PATH=/usr/local/staf/bin:$PATH
+export LD_LIBRARY_PATH=/usr/local/staf/lib:$LD_LIBRARY_PATH
+
+benchmark_dir="${currentDir}"
+echo "benchmark_dir: $benchmark_dir"
+
+echo "JDK_VERSION=${JDK_VERSION}"
+
+if [ "${JDK_VERSION}" = "8" ]; then
+    JDK_DIR=$(dirname $(dirname ${JAVA_BIN}))
+else
+	JDK_DIR=$(dirname ${JAVA_BIN})
+fi
+
+export JDK=$(basename ${JDK_DIR})
+echo "JDK=${JDK}"
+
+export JDK_DIR=$(dirname ${JDK_DIR})
+echo "JDK_DIR=${JDK_DIR}"
+
+
+######### Generated Script #########
+
+#TODO: Need to do some cleanup and restructure some files for adding other configs
+echo ""
+echo "********** START OF NEW TESTCI BENCHMARK JOB **********"
+echo "Benchmark Name: LibertyStartupDT Benchmark Variant: 17dev-4way-0-256-qs"
+echo "Benchmark Product: ${JDK}"
```

Comment:
`$JDK` is expected to be set in all our benchmark scripts. I do agree that this isn't very meaningful for now but since we aren't fully ready to enable Liberty perf tests on a regular basis. I had been thinking of creating an issue related to this and I just did: https://github.com/AdoptOpenJDK/openjdk-tests/issues/1144